src/Cli/dotnet/Installer/Windows/LocalizableStrings.Designer.cs

```diff
@@ -0,0 +1,63 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace Microsoft.DotNet.Cli.Installer.Windows {
+    using System;
+    
+    
+    /// <summary>
+    ///   A strongly-typed resource class, for looking up localized strings, etc.
```

Comment:
I'm not seeing this change in the original PR, why is it added here?

Reply:
oops. Sorry. Could you mind to approve this too?https://github.com/dotnet/sdk/pull/19082